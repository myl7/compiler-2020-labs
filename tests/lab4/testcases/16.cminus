float a(int x) {
    return x;
}

int b(float x) {
    return x;
}

void main(void) {
    outputFloat(a(1));
    output(b(1.1));
    return;
}
