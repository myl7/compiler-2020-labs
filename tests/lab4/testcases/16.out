1.000000
1
